recipe/meta.yaml

```diff
@@ -17,6 +17,7 @@ build:
 
 requirements:
   build:
+    - make
```

Comment:
This extra dep may create a new hash with the same build number and that can be confusing. Let's bump the bn here.

Reply:
Bumped the number and the builds passed.